src/app/components/Delegations/index.tsx

```diff
@@ -27,15 +27,18 @@ export const Delegations: FC<DelegationsProps> = ({
   const { t } = useTranslation()
 
   const tableColumns: TableColProps[] = [
-    { key: 'delegator', content: t('common.address') },
+    { key: 'delegator', content: linkType === 'validator' ? t('validator.title') : t('common.address') },
```

Comment:
`key: 'delegator'` is misleading (also in existing code in tableRows)

Ideally it would be specified like this
```ts
    { key: 'delegator', content: t('common.address'), hide: linkType === 'validator' },
    { key: 'validator', content: t('validator.title'), hide: linkType !== 'validator' },
```

Reply:
hide prop added